extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -279,6 +279,9 @@ export class AmpA4A extends AMP.BaseElement {
       height: this.element.getAttribute('height'),
     };
 
+    /** @private {?../../../src/layout-rect.LayoutRectDef} */
+    this.originalSlotSize_ = null;
```

Comment:
Seemed sensical to wait for onLayoutMeasure to ensure the values derived were correct (and its not required before then).